presto-hive/src/main/java/io/prestosql/plugin/hive/rubix/RubixConfig.java

```diff
@@ -86,6 +94,35 @@ public RubixConfig setCacheLocation(String location)
         return this;
     }
 
+    @MinDuration("0s")
```

Comment:
Could you also add tests for validation rules.
As here: https://github.com/prestosql/presto/blob/2fad53f05f32707abc54b6b384921365f1e87966/presto-oracle/src/test/java/io/prestosql/plugin/oracle/TestOracleConfig.java#L59

Reply:
Done